cmds/msr/msr.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// For now, this only works on linux.
+// +build linux
+
+// msr lets you read and write an msr for one or more cores.
+// The cores are specified via a filepath.Glob string.
+// Usage:
+// msr r glob 32-bit-msr-number
+// msr w glob 32-bit-msr-number 64-bit-value
+// For each MSR operation msr will print an error if any.
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+)
+
+const usage = `msr r glob register
+msr w glob register value`
+
+func main() {
+	var err error
+	a := os.Args[1:]
+
```

Comment:
Same with m below. (Side note: you can't edit comments on mobile? Thanks, GitHub.)